backend/src/main/java/gov/cdc/usds/simplereport/logging/AuditLoggingInstrumentation.java

```diff
@@ -49,7 +49,8 @@ public InstrumentationState createState() {
       state.setGraphqlDetails(
           new GraphQlInputs(
               parameters.getOperation(), parameters.getQuery(), parameters.getVariables()));
-      return new ExecutionResultContext(state);
+      _auditService.logGraphQlEvent(state, List.of());
+      return new SimpleInstrumentationContext<>();
```

Comment:
@benwarfield-usds One consequence of logging after execution is complete is that queries that fail holistic validation won't be logged, as the mechanism for enforcing such validation is to abort query execution at the validation stage.

It is possible to migrate enforcing modeled permissions to wrapped data fetchers in a `SchemaDirectiveWiring` class, similar to #1092, though this is less efficient and limits where the @requiredPermissions directive can be placed to just field and argument definitions. 

Not sure if the right thing to do here is to log queries prior to execution or to rewrite the modeled permissions enforcement, but it would help to know how important capturing the `errorPaths` property in the audit logs is.

Reply:
The alternate failure path for queries is to throw an `AbortExecutionException`, which the graphql-java query orchestrator captures [here](https://github.com/graphql-java/graphql-java/blob/v15.0/src/main/java/graphql/GraphQL.java#L501). This code path is pretty rare; it's used by [the max query depth validator](https://github.com/graphql-java/graphql-java/blob/master/src/main/java/graphql/analysis/MaxQueryDepthInstrumentation.java#L68), and I was hoping to use it here mainly because it makes testing easier. When this exception is thrown, the `onComplete` callback for execution instrumentation is not invoked. (The exception would be thrown [here](https://github.com/graphql-java/graphql-java/blob/v15.0/src/main/java/graphql/GraphQL.java#L493), skipping over the `onComplete` invocation [here](https://github.com/graphql-java/graphql-java/blob/v15.0/src/main/java/graphql/GraphQL.java#L496).)

Reply:
The transition from enforcing modeled permissions at the query level (rejecting the whole query with an error if the user's permissions don't permit the query as submitted) to enforcing them at the field and argument level (returning `null` and an error for specific fields that exceed the user's granted permissions) is complete. 

One upshot of the new approach is that no frontend changes are required. If a query is being used in multiple places in the frontend (e.g., to support the test queue and to support patient management) and requests data that exceeds what's available to entry_only users, those users will just have the extraneous fields nulled out.